source/testday/ussuri/milestone1.html.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: RDO Stein Test Day
+title: RDO Ussuri Test Day
 ---
 
-# Stein Test Day 1
+# Stein Milestone1 Test Day
```

Comment:
I guess that should be Ussuri Milestone1

Reply:
Good catch. Fixed!